Formula/wdt.rb

```diff
@@ -0,0 +1,50 @@
+class Wdt < Formula
+  desc "Transfer data as fast as possible over multiple TCP paths"
+  homepage "https://www.facebook.com/WdtOpenSource"
+  url "https://github.com/facebook/wdt.git", :revision => "bc22626deede683d11a2668121ebbd2d0389a7b8"
```

Comment:
Why not use one of the releases? They have actual tags and releases...

I want this formula too, but I don't want to deal with updating this thing everytime they push a commit.

Reply:
The latest release was from December 2016. It doesn't build against recent copies of folly. I can try to make it work but thought modern version might be more useful. I guess we could update it when and if there was a useful new feature/bugfix.

Reply:
Patches will be required upstream to remove the libfolly4wdt and resource folly so no releases can be used.